style.css

```diff
@@ -0,0 +1,75 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500&family=Poppins:wght@500&display=swap');
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+.main_section {
+  background: url('images/header-shapes\ mobile.svg') no-repeat center center;
+  background-color: #fff;
+  max-width: 375px;
+  height: 100vh;
+}
+
+.material-icons {
+  color: #6070ff;
+  font-size: larger;
+  font-weight: bolder;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 9%;
+  margin-left: -20px;
+}
+
+ul {
+  display: flex;
+  list-style: none;
+  margin-left: -20px;
+}
+
+h1 {
+  margin-top: 58px;
+  font-size: 50px;
+  font-weight: bolder;
+  color: #2e394b;
+  text-align: justify;
+  line-height: normal;
+}
```

Comment:
- [ ] [OPTIONAL] Please use the same color as the one in the Figma design, it's very important to stick to the design this will teach us to follow the client's requirement in our future job 😃 
the correct color is `#172B4D`